pkgs/by-name/dy/dyalog/package.nix

```diff
@@ -149,9 +180,6 @@ stdenv.mkDerivation (finalAttrs: {
     for exec in "dyalog" "dyalog.rt"; do
         patchelf ${dyalogHome}/$exec --add-needed libncurses.so
     done
-  ''
-  + lib.optionalString htmlRendererSupport ''
-    patchelf ${dyalogHome}/libcef.so --add-needed libudev.so --add-needed libGL.so
```

Comment:
Are we sure we can remove libGL.so from here?

I'm getting the following error even just by entering the interpreter:
`
ERR: Display.cpp:1079 (initialize): ANGLE Display::initialize error 12289: Could not dlopen libGL.so.1: libGL.so.1: cannot open shared object file: No such file or directory
`

It doesn't actually cause the HTMLRenderer not to work, just shows the error upon launch.

Reply:
I got the same thing. It went away after rebooting. I made sure to have nix-ld disabled. Does it persist for you?